app/src/main/java/io/github/djunicode/canteenapp/RetrofitInterfaces/ApiInterface.java

```diff
@@ -0,0 +1,15 @@
+package io.github.djunicode.canteenapp.RetrofitInterfaces;
+
+import java.util.List;
+
+import io.github.djunicode.canteenapp.models.MenuItem;
+import retrofit2.Call;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+
+public interface ApiInterface {
+
```

Comment:
Why are there two interface files? Arnav had created LoginInterface and this is another interface. Any specific reason for this? You could just add all the endpoints in a single interface file. Do one thing, pull the current upstream code, merge the login endpoint in this interface (You may need to refactor the code at other places too), rebase with the development branch and then push the changes.

Reply:
I had merged the development branch before pushing . 
Actually arnav had named the interface loginInterface so i thought he wanted to keep it separate 

Also the i had to make a singleton of retrofit 
Because the base activity things didn't work well with fragments

Reply:
So y'all are thinking of having two different interfaces?

Reply:
No i spoke with arnav and we decided to keep a single interface 

I will merge both of them today and push again

Reply:
Cool :+1: